config/init.lua

```diff
@@ -104,6 +105,10 @@ for _, key in pairs(keys) do
     config.register_key(key)
 end
 
+function cleanup_background()
+  os.execute("pkill way-cooler-bg")
+end
+
 
 -- Execute some code after Way Cooler is finished initializing
 function way_cooler_init()
```

Comment:
Okay, I totally missed that. Those changes were added in another PR, I'm gonna leave that alone. If we want to look at fixing this later I'll open another PR.